theme-dev/Manifest.md

```diff
@@ -0,0 +1,21 @@
+<h1 align="center">Manifest</h1>
+
+**DO NOT CHANGE THE JSON FILE UNLESS NECESSARY, OR ELSE YOU WILL FUCK UP YOUR THEME BECAUSE YOU DON'T KNOW WHAT YOU'RE DOING.**
```

Comment:
```suggestion
**DO NOT CHANGE THE JSON FILE UNLESS NECESSARY, OR ELSE YOU WILL MESS UP YOUR THEME BECAUSE YOU DON'T KNOW WHAT YOU'RE DOING.**
```

Reply:
What's the best way to tell someone you will break something, by swearing 
I am not making my docu too professional

Reply:
Yes you will fuck up your teme if you dont know what you';re doing